tools/tensorflow_docs/api_generator/parser.py

```diff
@@ -386,6 +386,8 @@ def _gen_pairs(items):
     The original items, in pairs
   """
   assert len(items) % 2 == 0
+  if not items:
+    return
```

Comment:
How about this instead to iter?

@MarkDaoust 
```
zip(items[::2], items[1::2])
```

Reply:
This looks much cleaner! Thx.